<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MPINDO02PC560" ignored="0" name="ValidationHeader_ResidentialSection" tests="1" failures="1" timestamp="14 Sep 2018 10:53:34 GMT" time="329.611" errors="0">
  <testcase name="@AfterClass quitReport" time="0.084" classname="Ebiz.BaseClass">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at SupportLibraries.report.FailTest(report.java:169)
at Ebiz.BaseClass.quitReport(BaseClass.java:92)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass quitReport -->
  <testcase name="ValidationHeader_ResidentialSection1" time="315.237" classname="Ebiz.Entrymdoule.ValidationHeader_ResidentialSection">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at SupportLibraries.report.FailTest(report.java:149)
at ObjectRepository.homePage.ValidateResiCategoryHeaderLinks(homePage.java:1484)
at Ebiz.Entrymdoule.ValidationHeader_ResidentialSection.ValidationHeader_ResidentialSection1(ValidationHeader_ResidentialSection.java:68)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- ValidationHeader_ResidentialSection1 -->
</testsuite> <!-- ValidationHeader_ResidentialSection -->
